tendrl/commons/flows/base_flow.py

```diff
@@ -54,17 +56,8 @@ def run(self):
         # TODO(rohan) make a func outta the below logging common code
         msg = "Processing pre-runs for flow: %s" % self.job['run']
         LOG.info(msg)
```

Comment:
Line no 67, 88, 100, 120 are setting the log messages but they are not written immediately to etcd. Why so?